src/components/Reports/WbsTable/index.js

```diff
@@ -1 +1,2 @@
-export { WbsTable } from './WbsTable';
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Please fix this error, not using "eslint-disable-next-line"

Reply:
fixed